test/integration/policy_comp_operator_test.go

```diff
@@ -19,6 +19,8 @@ import (
 	"github.com/stolostron/governance-policy-framework/test/common"
 )
 
+const ComplianceScanTimeoutSeconds = 900
```

Comment:
Great minds think alike? I just opened https://github.com/stolostron/governance-policy-framework/pull/506 to increase the "max" timeout to 15 minutes. I'm fine if we want to keep the timeout for this test separate, although I think it is the main use of the "max" timeout...